cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -221,13 +255,19 @@ fn is_redundant_move<I: VCodeInst>(insn: &I) -> bool {
     }
 }
 
+/// Is this type a reference type?
+fn is_reftype(ty: Type) -> bool {
```

Comment:
nano-nit: since most of our users will be coming at this on 64-bit targets, maybe put that first in the disjunction?  That way we can save perhaps 1/3 of a nanosecond per call :-)